File: ggp-base/src/org/ggp/base/util/gdl/grammar/Gdl.java
package org.ggp.base.util.gdl.grammar

import java.io.ObjectStreamException
import java.io.Serializable

def abstract class Gdl implements Serializable


    def abstract bool isGround()

    def abstract String toString()

	/**
	 * This method is used by deserialization to ensure that Gdl objects
	 * loaded from an ObjectInputStream or a remote method invocation
	 * are the versions that exist in the GdlPool.
	 */
    protected Object readResolve() throws ObjectStreamException 
        return GdlPool.immerse(this)

File: ggpy/cruft/autocode/Gdl.py
#!/usr/bin/env python
""" generated source for module Gdl """
# package: org.ggp.base.util.gdl.grammar
import java.io.ObjectStreamException

import java.io.Serializable

@SuppressWarnings("serial")
class Gdl(Serializable):
    """ generated source for class Gdl """
    def isGround(self):
        """ generated source for method isGround """

    def __str__(self):
        """ generated source for method toString """

    # 
    # 	 * This method is used by deserialization to ensure that Gdl objects
    # 	 * loaded from an ObjectInputStream or a remote method invocation
    # 	 * are the versions that exist in the GdlPool.
    # 	 
    def readResolve(self):
        """ generated source for method readResolve """
        return GdlPool.immerse(self)

